### DISCUSSION TITLE
TP4 - Ejercicio 7

### DISCUSSION THREAD

**Discussion by Mirkopoj** [INGCOM-UNRN-P1/cursada-2023#128] [Preguntas y respuestas] [answered]
En el ejercicuo se pide buscar el largo seguro, que dependía de tener un limite de caracteres para terminar la busqueda si la cadena no tenia su caracter de terminacion.

En el prototipo que se nos da, no hay ningun parametro para el largo unicamente un puntero a la cadena. ¿Habria que agregar el parametro con el limite de la busqueda o buscar otra forma de limitar en el caso de que la cadena no tenga terminación?

**Comment by martinvilu** [comment, marked as answer]
Si, efectivamente esta faltando la 'capacidad' de la cadena para ser considerada segura.
